services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/controller/SubnetController.java

```diff
@@ -537,7 +539,11 @@ public ResponseId deleteSubnetState(@PathVariable String projectId, @PathVariabl
             }
 
             // delete subnet routing rule in route manager
-            this.subnetService.deleteSubnetRoutingRuleInRM(projectId, subnetId);
+            try {
+                this.subnetService.deleteSubnetRoutingRuleInRM(projectId, subnetId);
```

Comment:
This method was to delete the entire routing table associated with this subnet in RM. Let us rename it to deleteeSubnetRoutingTable.